extensions/control-plane/api/management-api/contract-definition-api/src/main/java/org/eclipse/edc/connector/api/management/contractdefinition/model/ContractDefinitionUpdateDtoWrapper.java

```diff
@@ -0,0 +1,65 @@
+/*
+ *  Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.connector.api.management.contractdefinition.model;
+
+import java.util.Objects;
+
+public class ContractDefinitionUpdateDtoWrapper {
```

Comment:
i like this idea, in fact I might even adopt this into the Asset/DataAddress subfeature, but since the sole purpose of this class is to wrap the dto and the ID, personally I would make it package private, or maybe even an inner class of the `ContractDefinitionApiController`, if possible. In case a smidge of documentation wouldn't hurt :)

Reply:
I like this too